jbosstools/multiple/jbosstools-multiple.target

```diff
@@ -88,6 +79,15 @@
       <unit id="org.slf4j.api" version="1.7.2.v20121108-1250"/>
     </location>
 
+    <!-- JBIDE-12815 JBoss Tools Locus bundles: not quite ready to be included -->
+    <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+        <repository location="http://repository.jboss.org/nexus/content/unzip/unzip/org/jboss/tools/locus/update.site/1.1.1.Final/update.site-1.1.1.Final.zip-unzip/"/>
+        <!-- JBIDE-12971 JBIDE-19110 Mockito -->
+        <unit id="org.jboss.tools.locus.mockito" version="1.9.5.Final_patched_TEIIDDES-1681-v20131024-0922-B58"/>
+        <!-- JBIDE-12972 Fest Assert
+        <unit id="org.jboss.tools.locus.easytesting.fest-assert" version="1.4.0.Final-v20130507-2028-B7"/> -->
```

Comment:
To be clear this is *also* adding Fest Assert not just Mockito.

I think its fine to get it in as long as just used in testing but only if someone actually wants it.